packages/core/src/render3/instructions/property_interpolate.ts

```diff
@@ -0,0 +1,429 @@
+import {getSelectedIndex} from '../state';
+
+import {interpolation1, interpolation2, interpolation3, interpolation4, interpolation5, interpolation6, interpolation7, interpolation8, interpolationV} from './instructions';
+import {TsickleIssue1009} from './property';
+import {elementPropertyInternal} from './shared';
+
+
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * Shared reference to a string, used in `propertyInterpolate`.
+ */
+const EMPTY_STRING = '';
+
+/**
+ *
+ * Update an interpolated property on an element.
+ *
+ * Used when the value passed to a property has 1 interpolated value in it, an no additional text
+ * surrounds that interpolated value:
+ *
+ * ```html
+ * <div title="{{v0}}"></div>
+ * ```
+ *
+ * Should result in this usage:
+ *
+ * ```ts
+ * propertyInterpolate('title', v0);
+ * ```
+ *
+ * If the property name also exists as an input property on one of the element's directives,
+ * the component property will be set instead of the element property. This check must
+ * be conducted at runtime so child components that add new `@Inputs` don't have to be re-compiled.
+ *
+ * @param propName The name of the property to update
+ * @param prefix Static value used for concatenation only.
+ * @param v0 Value checked for change.
+ * @param suffix Static value used for concatenation only.
+ * @returns itself, so that it may be chained.
+ */
+export function propertyInterpolate(propName: string, v0: any): TsickleIssue1009 {
+  propertyInterpolate1(propName, EMPTY_STRING, v0, EMPTY_STRING);
+  return propertyInterpolate;
+}
+
+
+/**
+ *
+ * Update an interpolated property on an element.
+ *
+ * Used when the value passed to a property has 1 interpolated value in it:
+ *
+ * ```html
+ * <div title="prefix{{v0}}suffix"></div>
+ * ```
+ *
+ * Should result in this usage:
```

Comment:
```suggestion
 * Its compiled representation is:
```

`should` implies that it could be different. (Change in all subsequent instances as well)

Reply:
Made in all doc comments.